File: Sorting/bubbleSort/bubble_sort.cpp
#include <iostream>
using namespace std;
#define for_i0n for(int i=0;i<n;i++)
#define for_ji1n for(int j=0;j<n - i - 1;j++) 
int main() {
  int array[] = {4,1,9,6,2};
  // bubble sort
  
  for_i0n {
    for_ji1n {
      if(array[j] > array[j + 1]) {
        int temp = array[j];
        array[j] = array[j + 1];
        array[j + 1] = temp;
      }
    }
  }
  
  cout<<"Updated Array : "<<endl;
  for(int i=0;i<n;i++) {
    cout<<array[i]<<" ";
  }
  cout<<endl;
  return 0;
}

File: Sorting/bubbleSort/bubble_sort.js
function bubble_sort(arr)
{
    for (let i = 0; i < arr.length; i++)
    {
        for (let j = i; j < arr.length; j++)
        {
            if (arr[i] > arr[j])
            {
                let temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }

    return arr;
}

let arr = [3, 1, 2, 5, 4];

console.log(bubble_sort(arr));
